### PR TITLE
EAPI2 Again

### PR DESCRIPTION
**Pull request by flappyports**
@SoapGentoo please review

### CODE REVIEW

File: media-gfx/jpegtoavi/jpegtoavi-1.5.ebuild [hunk 1]
```diff
@@ -13,17 +13,15 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86"
 IUSE=""
 
-src_prepare() {
-	epatch "${FILESDIR}"/${P}-Makefile.patch
-}
+PATCHES=( "${FILESDIR}/${P}-Makefile.patch" )
```
Comment (Zlogene) [inline media-gfx/jpegtoavi/jpegtoavi-1.5.ebuild]: drop it and do everything from prepare, including CC respect

File: app-text/tokyodystopia/tokyodystopia-0.9.15.ebuild [hunk 2]
```diff
@@ -17,28 +15,27 @@ IUSE="examples"
...
+)
 
 src_configure() {
-	econf --libexecdir=/usr/libexec/${PN} || die
+	econf --libexecdir="${EPREFIX}"/usr/libexec/${PN}
 }
 
 src_install() {
-	emake DESTDIR="${D}" install || die "Install failed"
+	HTML_DOCS=( doc/* )
```
[the remark this answers is not in the document]
Comment (flappyports) [reply app-text/tokyodystopia/tokyodystopia-0.9.15.ebuild]: I know I can use dodoc -r for the HTML, but why not let the default call do it?

[the remark this answers is not in the document]
Comment (Zlogene) [reply app-text/tokyodystopia/tokyodystopia-0.9.15.ebuild]: but what's the point? these arrays are useful if you do not define src_install and put it somewhere above

File: app-emulation/coldfire/coldfire-0.3.1.ebuild [hunk 3]
```diff
@@ -17,13 +15,12 @@ IUSE=""
 DEPEND="sys-libs/ncurses
```
Comment (Zlogene) [inline app-emulation/coldfire/coldfire-0.3.1.ebuild]: slotting?

Comment (Zlogene) [reply app-emulation/coldfire/coldfire-0.3.1.ebuild]: (for both)

File: app-text/tokyodystopia/tokyodystopia-0.9.15.ebuild (hunk 2 above)
[the remark this answers is not in the document]
Comment (flappyports) [reply app-text/tokyodystopia/tokyodystopia-0.9.15.ebuild]: I call HTML_DOCS inside of src_install so globbing will work.  Calling it in global scope will break it.

File: app-emulation/coldfire/coldfire-0.3.1.ebuild (hunk 3 above)
Comment (flappyports) [reply app-emulation/coldfire/coldfire-0.3.1.ebuild]: fixed

File: app-text/tokyodystopia/tokyodystopia-0.9.15.ebuild (hunk 2 above)
[the remark this answers is not in the document]
Comment (SoapGentoo) [reply app-text/tokyodystopia/tokyodystopia-0.9.15.ebuild]: also, do `HTML_DOCS=( doc/. )`, the glob is unnecessary

File: app-text/tokyodystopia/tokyodystopia-0.9.15.ebuild [hunk 4]
```diff
@@ -17,28 +15,27 @@ IUSE="examples"
...
 
-	dohtml doc/* || die
+	default
 
 	if use examples; then
 		insinto /usr/share/${PF}/example
-		doins example/* || die "Install failed"
+		doins example/*
```
Comment (SoapGentoo) [inline app-text/tokyodystopia/tokyodystopia-0.9.15.ebuild]: `doins -r example/.` - avoid unnecessary globs

File: media-gfx/jpegpixi/jpegpixi-1.1.1.ebuild [hunk 5]
```diff
@@ -15,7 +15,7 @@ IUSE=""
 DEPEND="virtual/jpeg"
 
 src_install() {
-	dobin jpegpixi jpeghotp || die
-	doman man/jpegpixi.1 man/jpeghotp.1
+	dobin jpeg{hotp,pixi}
+	doman man/jpeg{hotp,pixi}.1
 	dodoc AUTHORS NEWS README README.jpeglib ChangeLog
```
Comment (SoapGentoo) [inline media-gfx/jpegpixi/jpegpixi-1.1.1.ebuild]: `einstalldocs` here

File: media-gfx/jpegtoavi/jpegtoavi-1.5.ebuild [hunk 6]
```diff
@@ -13,17 +13,15 @@ SLOT="0"
...
 
 src_compile() {
 	tc-export CC
-	emake || die
+	default
 }
```
Comment (SoapGentoo) [inline media-gfx/jpegtoavi/jpegtoavi-1.5.ebuild]: rather override `src_configure`, then you don't have to call `default`

File: media-libs/celt/celt-0.5.1.3.ebuild [hunk 7]
```diff
@@ -22,8 +22,6 @@ src_configure() {
 }
 
 src_install() {
-	emake DESTDIR="${D}" install || die "emake install failed."
-	dodoc README TODO || die "dodoc failed."
-
-	use static-libs || find "${D}" -name '*.la' -delete
+	default
+	use static-libs || (find "${D}" -name '*.la' -delete || die "Pruning failed")
```
Comment (SoapGentoo) [inline media-libs/celt/celt-0.5.1.3.ebuild]: this is hard to read, split it up in a proper `if ... then; fi` block

File: media-libs/tse3/files/tse3-0.2.7-gcc4.patch [hunk 8]
```diff
@@ -1,5 +1,5 @@
---- ./src/tse3/cmd/Phrase.h~	2002-07-28 17:17:11.000000000 +0200
-+++ ./src/tse3/cmd/Phrase.h	2005-07-29 15:45:03.000000000 +0200
+--- a/src/tse3/cmd/Phrase.h	2002-07-28 17:17:11.000000000 +0200
++++ b/src/tse3/cmd/Phrase.h	2005-07-29 15:45:03.000000000 +0200
```
Comment (SoapGentoo) [inline media-libs/tse3/files/tse3-0.2.7-gcc4.patch]: also, in general remove the dates too, the just increase the noise in patches and arent useful

File: media-libs/tse3/tse3-0.3.1-r1.ebuild [hunk 9]
```diff
@@ -17,17 +17,21 @@ IUSE="alsa oss"
...
+	"${FILESDIR}/${P}+gcc-4.3.patch"
+)
+
 src_prepare() {
+	default
+	mv configure.in configure.ac || die "Moving configure.in -> .ac failed"
 	if use oss; then
-		cp "${WORKDIR}"/awe_voice.h src/
+		cp "${WORKDIR}"/awe_voice.h src/ || die "copy failed"
 		append-flags -DHAVE_AWE_VOICE_H
```
Comment (SoapGentoo) [inline media-libs/tse3/tse3-0.3.1-r1.ebuild]: try making this `append-cppflags` instead (this should be giving you a warning anyhow)

**Pull request closed without merging**